### ISSUE TITLE
Configuration resets after restart

### ISSUE THREAD

**Issue by Ben1asdfas** [deepch/RTSPtoWeb#398]
I created a service which starts RTSPtoWeb after restart of linux.

My problem is, every time linux restarts, the configuration resets.

Can someone explain a common solution for me as a non linux expert?

**Comment by Ben1asdfas** [comment]
Hi HurtZMH, 

thanks for your quick reply ! I intalled RTSPtoWeb as described on the installation guide - as a docker container on debian 12. I did the configuration only on the RTSP webinterface, but it does not seem to safe it right to the config file.

Now i tried to directly edit the config.json in the RTSPtoWeb folder and now it works - means that when i restart the rtsp service or the whole linux system it now loads my configuration right.

**Comment by HurtZMH** [comment]
Right! So it's permission problem, to work normally, including the streams by web, you should look if the user has permission to change this file.
